include/cpr/session.h

```diff
@@ -112,6 +113,17 @@ class Session : public std::enable_shared_from_this<Session> {
     void SetAcceptEncoding(AcceptEncoding&& accept_encoding);
     void SetLimitRate(const LimitRate& limit_rate);
 
+    /**
+     * Returns a reference to the content sent in previous request
+     */
```

Comment:
```suggestion
    /**
      * Returns a reference to the content sent in previous request.
      **/
```

That way its the same as for all other comments. I know I need a linter for it at some point ;)